codegenerator/cli/templates/static/codegen/src/ink/components/Messages.res

```diff
@@ -0,0 +1,33 @@
+open Belt
+open Ink
+module Message = {
+  @react.component
+  let make = (~message: CustomHooks.InitApi.message) => {
+    <Text
+      color={switch message.kind {
+      | Warning => Secondary
+      | Info => Info
+      | Destructive => Danger
+      }}>
+      {message.content->React.string}
+    </Text>
+  }
+}
+
+@react.component
+let make = (~config) => {
+  let messages = CustomHooks.useMessages(~config)
+  <>
+    {switch messages {
+    | Data(messages) =>
+      <>
+        {messages
+        ->Array.mapWithIndex((i, message) => {<Message key={i->Int.toString} message />})
+        ->React.array}
+      </>
+    | Loading => React.null //Don't show anything while loading
+    | Err(_) =>
+      <Message message={kind: Destructive, content: "Failed to load messages from envio server"} />
+    }}
```

Comment:
Do we actually want to show that there's a failure? It's not important to display if there is but my only concern is it would go by undetected as it would present the same as no messages at all.

Reply:
Maybe we should implement something like `envio.dev/app/hyperindex/send-error` and integrate it with Sentry?